smoketest/graphql2rotations_test.go

```diff
@@ -0,0 +1,153 @@
+package smoketest
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/target/goalert/smoketest/harness"
+	"testing"
+)
+
+func TestGraphQL2RotationsFavorite(t *testing.T) {
```

Comment:
Just call it `TestGraphQLFavoriteRotations` also update the filename to match the test name.

Putting favorite first will group all the favorite gql tests together.

The `2` bit is only temporary while we transition to the new library.